etc/scripts/openhub_scraper.py

```diff
@@ -0,0 +1,36 @@
+# A script to extract openhub license names, their urls, and save it in a
+# json file
+import json
+import urllib2
+from bs4 import BeautifulSoup
+
+total_pages = 17
+license_dict = {}
+license_list = []
+
+
+def extract_openhub_licenses():
+    try:
+        for i in range(1, total_pages + 1):
+            url = 'https://www.openhub.net/licenses?page=' + str(i)
+            page = urllib2.urlopen(url)
+            parsed_page = BeautifulSoup(page, 'html.parser')
+            all_licenses = parsed_page.find(id='license').select(
+                'table.table-condensed.table-striped.table')
+            license_rows = all_licenses[0].find_all('a', href=True)
+
+            for license in license_rows:
+                license_dict['openhub_url'] = license['href']
+                license_dict['name'] = license.get_text()
+                license_list.append(license_dict.copy())
+
+        with open('openhub_licenses.json', 'w') as f:
+            f.write(json.dumps(license_list, indent=4,
+                               ensure_ascii=False).encode('utf8'))
+
+    except Exception as e:
```

Comment:
Just do not try/except here. Let exceptions bubble up: you are hiding the stack trace otherwise

Reply:
dumps should return UTF alright. why encode?

Reply:
Addressed !